buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -367,14 +367,14 @@ public final class {{class}} {
 
   /**
    * Kind of HTTP protocol used.
-   * @deprecated This item has been removed as of 1.20.0 of the semantic conventions.
+   * @deprecated This item has been removed as of 1.20.0 of the semantic conventions. There is no replacement.
```

Comment:
I think can use `network.protocol.version` instead

Reply:
Ah, I found it in the changelog:
```
| `http.flavor` | string | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Deprecated, use `network.protocol.name` instead. | `1.0` |
```
I'll change it to `NETWORK_PROTOCOL_NAME` then :)